src/src/Commands/Tasks.cpp

```diff
@@ -51,8 +51,13 @@ String Command_Task_Disable(struct EventStruct *event, const char *Line)
   unsigned int varNr;
   String dummy;
 
-  if (validTaskVars(event, taskIndex, varNr) && (PluginCall(PLUGIN_EXIT, event, dummy) || true) && setTaskEnableStatus(taskIndex, false)) {
-    return return_command_success();
+  if (validTaskVars(event, taskIndex, varNr)) {
+    PluginCall(PLUGIN_EXIT, event, dummy);
+    if (setTaskEnableStatus(taskIndex, false)) {
+      return return_command_success();
+    } else {
+      return return_command_failed();
```

Comment:
This else can be deleted as the default return is also to `return_command_failed()`

Reply:
Fixed now.